### ISSUE TITLE
1px gray border on top and bottom on Steam Deck

### ISSUE THREAD

**Issue by jbaiter** [Albeoris/Memoria#789]
When playing FF9 with Memoria and Moguri on Steam Deck, there's a 1px gray (`#131517`) border on the top and the bottom of the screen. It's not really noticeable when there's a bright background, but during scene changes and dark scenes, it's really distracting.

See these screenshots (view using dark theme or use an image editor):
![377840_20240825212640_1](https://github.com/user-attachments/assets/dac9d03e-fff5-4770-925e-49ec79a81d13)
![377840_20240825212810_1](https://github.com/user-attachments/assets/f21ec684-2d3e-4ea0-a28a-aaf41ad775dc)
![377840_20240825212832_1](https://github.com/user-attachments/assets/517eabd0-4865-4527-9c5f-7ffe44c482fb)

**Comment by barkermn01** [comment]
Unfortunately that's not in the screenshot would you be able to take a picture of it on your device, so we can see it?

**Comment by jbaiter** [comment]
It is in the screenshots, just not very well visible in the GitHub style. Download the PNGs, open them in an image viewer and zoom in, it's there.

**Comment by snouz** [comment]
I couldn't initially open the images, now I can. I see it from your image

![image](https://github.com/user-attachments/assets/e5ec917e-3329-47c8-a7f0-3ce741a53254)

I thought this was the same issue as #572, but it seems to happen in non-widescreen parts of your screenshots

**Comment by jbaiter** [comment]
I think it's indeed the same bug, the line is not only in the non-widescreen parts, it's just more visible there due to to the increased contrast. If you zoom into the second image, you can see that it's present on the bottom of the stage, too:

![zoomed](https://github.com/user-attachments/assets/49c310f9-18da-4f56-87ad-fd7197a1a21e)

**Comment by snouz** [comment]
No idea if we'll be able to fix it and what effort it will require, or if that's also a vanilla bug, but is there noticeable/bothering on your screen? I can kind of see borders, but my screen is large enough that it's extremely negligeable.

**Comment by jbaiter** [comment]
Yeah, it's very noticeable on the Steam Deck OLED at least, probably due to the per-pixel brightness of the OLED screen :-/ Not a complete dealbreaker, but it definitely detracts from the experience.

**Comment by canesalato** [comment]
I can confirm the bug in my steam deck OLED. It is quite distracting, but more than anything i am worried about burn-in. Also happens when playing on my LG OLED tv, at 16/9 4k

**Comment by SamsamTS** [comment]
This is not limited to steam deck.
My guess is it's that texture that get compressed into 1px and therefore looks gray. Difficult to see on my 2K monitor but on my 1080p monitor it's very visible.

![image](https://github.com/user-attachments/assets/160d8068-c537-457e-872e-cb9f4ea9fb5f)
